organization/_posts/2021/2021-05-06-coordination.md

```diff
@@ -0,0 +1,166 @@
+---
+title: "HSF Coordination Meeting #207, 6 May 2021"
+layout: plain_toc
+---
+
+Present:
+Allison Hall,
+Andreas Salzburger,
+Ben Morgan,
+Caterina Doglioni,
+David Lange,
+Eduardo Rodrigues,
+Graeme Stewart,
+Josh McFayden,
+Kilian Lieret,
+Krzysztof Genser,
+Kyle Knoepfel,
+Liz Sexton-Kennedy,
+Mason Proffitt,
+Meirin Oan Evans,
+Michel Jouvin,
+Michel Villanueva,
+Paul Laycock,
+Serhan Mete,
+Stefan Roiser,
+Sudhir Malik,
+Teng Jian Khoo
+
+Apologies: Andrea Valassi, Dorothea vom Bruch
+  
+## News, general matters, announcements
+
+### HL-LHC Review Phase 2
+
+We have had two mini-workshops that presented plans and ideas for the HL-LHC era from both experiments and projects (a *conversation*):
+
+* Detector Simulation, Thursday 29 April <https://indico.cern.ch/event/1028379/>
+* Analysis, Tuesday 4 May <https://indico.cern.ch/event/1028381/>
+
+These were well attended and extremely useful.
+
+The inputs and discussions will inform the documents that are being written. First drafts to be ready **30 June**, so that they can be discussed and get feedback over the summer.
+
+### Google Summer of Code
+
+GSoC administrators have requested slots for all proposals where they received a good proposal (~27).
+
+### Proposal for DUNE Framework Requirements Review
+
+Framework requirements review kick-off to be held next week (probably Mon. or Tues., time to be settled by the end of the week).  Review mini-workshop will be during the week of May 24th or May 31st.
+
+## Working Group Updates
+
+### Data Analysis
+
+Second instance of "Wider HEP/NP analysis community" meetings yesterday. Agenda with recording & live notes [here](https://indico.cern.ch/event/1033484/).
+
+* Excellent, interesting talks from FAIR/GSI, LZ, Xenon
+* Highlights:
+    * Sharing of basic SW framework and dynamic computing resource allocation at GSI
+    * Effective Gitlab CI, common analysis SW in LZ
+    * 100% python framework at Xenon
+* Put together our own "talking points"
+    * DUNE framework review -- generalisable?
+    * How have (some, smaller?) experiments managed to converge on one analysis framework
+    * Keen to revisit discussion on recognition (always a popular topic!)
+
+Need to define agenda for next instance, was for 19 May, but perhaps we should postpone a week to avoid vCHEP...
```

Comment:
```suggestion
Need to define an agenda for the next instance, currently scheduled for 19 May but perhaps we should postpone it to a week later to avoid vCHEP...
```